### ISSUE TITLE
[Bug Report]: "Attempted to send plugin message to CraftPlayer{name=darkshabby}, but it could not be sent."while kicking someone

### ISSUE THREAD

**Issue by wksdjfhgfrejwksld** [A248/LibertyBans#287]
### LibertyBans Version

LibertyBans 1.1.0-RC2

### I have confirmed that ...

- [X] LibertyBans is up to date
- [X] No similar issue has been reported

### Platform

Spigot/Paper, Velocity

### Description

message below is reported while kicking someone in server console:
[space.arim.libertybans.core.env.AbstractEnvEnforcer] Attempted to send plugin message to CraftPlayer{name=darkshabby}, but it could not be sent. This suggests you enabled 'use-plugin-messaging' but are not using a network. Please address this critical security flaw immediately. It leaves your server vulnerable to clients spoofing the plugin messaging channel.
but i am really using velocity(Velocity 3.4.0-SNAPSHOT (git-1db8c8c6-b458)) and followed https://docs.libertybans.org/#/Running-Multiple-Instances to install this plugin

**Comment by wksdjfhgfrejwksld** [comment]
plugins on backend installed:
LibertyBans, LuckPerms, packetevents, PlaceholderAPI, PlayerChat, PlayerMenu, PlugManX, ProtocolLib, ServersNPC, ViaBackwards,ViaRewind, ViaRewind-Legacy-Support, ViaVersion

**Comment by wksdjfhgfrejwksld** [comment]
> Hello! Are you using a proxy? If yes, does it have plugin messaging support?

yes,im using velocity as proxy and other plugin that requied plugin messaging works

**Comment by A248** [comment]
Please update to the latest development version. This bug was fixed in development versions, but unfortunately I didn't have a chance to make an official release because my computer is in for repairs.
